### DISCUSSION TITLE
Dags Run report from Airflow UI

### DISCUSSION THREAD

**Discussion by Praful19** [apache/airflow#21782] [Q&A]
Hi All,
 I wondering if there is any way where  we can pull report of Dags run time from airflow UI ? like if i have 10 dags running I want to see start time, end time of Dag run. and time it took to complete the Dags. One way is querying database but I am looking if we can pull the report for Airflow UI ?

Regards
Praful

**Comment by potiuk** [comment]
Use REST API https://airflow.apache.org/docs/apache-airflow/stable/stable-rest-api-ref.html
